### ISSUE TITLE
Plugin does not send notification when a build flow job starts

### ISSUE THREAD

**Issue by olhado** [jenkinsci/slack-plugin#22]
I have a a job that is using the build flow plugin to run several jobs in parallel. While aborts and build complete notifications are sent, the initial job started notification never is sent to slack, it appears. All the child jobs send notifications that they were started by the build flow job.

**Comment by peergum** [comment]
This should work in the coming release.

**Comment by samrocketman** [comment]
Release date: https://github.com/jenkinsci/slack-plugin/milestones

**Comment by samrocketman** [comment]
Slack Plugin 1.7 has been released.  This a fix for this has been included in it.  Please try out new version and if it doesn't resolved this issue then ask for this to be re-opened and provide more details.

**Comment by 256x240** [comment]
Hi! I have this bug now. How to solve this problem?

**Comment by dmugtasimov** [comment]
please, see, new issue here https://github.com/jenkinsci/slack-plugin/issues/140
